tests/integration/cli/utils.py

```diff
@@ -142,6 +142,7 @@ def wrapper(f):
         f = pytest.mark.xfail(
             strict=False, reason="GitHub rate limiting issues or plugin install issues"
         )(f)
+        f = pytest.mark.manual()(f)
```

Comment:
@mikeshultz An alternative to the skip idea here https://github.com/ApeWorX/ape/pull/1890

Reply:
I can't seem to find docs on this, can you share?  Seems more useful than a strict skip.

Reply:
Is there an alternative to this that would require explicit action to run rather than explicit action to skip?

Reply:
rebase after your work!